jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -2672,6 +2672,65 @@ ecma_op_species_constructor (ecma_object_t *this_value, /**< This Value */
 
 #endif /* ENABLED (JERRY_ES2015) */
 
+/**
+ * 7.3.18 Abstract operation Invoke when property name is a magic string
+ *
+ * @return ecma_value result of the invoked function or raised error
+ *         note: returned value must be freed with ecma_free_value
+ */
+extern inline ecma_value_t JERRY_ATTR_ALWAYS_INLINE
```

Comment:
Do we need the `extern` here?

Reply:
IMO that corresponds to some kind of Windows port issue. cc @ossy-szeged.

Reply:
Extern is needed only if the inline function is called outside of jerry-core, typically from unit-core.
It seems it isn't that case, it should be safe to remove the extern keyword.